### PR TITLE
Add SRID Enum

### PR DESCRIPTION
**Pull request by Riley19280**
Adds and SRID enum, helpful for referencing the spatial systems, instead of memorizing magic numbers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ $londonEye = Place::create([
 
 $whiteHouse = Place::create([
     'name' => 'White House',
-    'location' => new Point(38.8976763, -77.0365298, 4326), // with SRID
+    'location' => new Point(38.8976763, -77.0365298, SRID::GCS->value), // with SRID
```
Comment (MatanYadaev) [inline README.md:109]: An import of the `SRID` enum is missing in the code block.

File: src/Constants/SRID.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MatanYadaev\EloquentSpatial\Constants;
+
+/**
+ * Spatial reference identifiers identify the type of coordinate system to use.
+ *
+ * This includes the most common ones listed on wikipedia
+ * Sourced from https://en.wikipedia.org/wiki/Spatial_reference_system
```
Comment (MatanYadaev) [inline src/Constants/SRID.php:9]: Please correct me if I'm wrong, but the Wikipedia article didn't mention that these SRIDs are the most common ones.
As far as I know, 4326 and 3857 are the most common SRIDs. https://doc.arcgis.com/en/insights/latest/analyze/spatial-references.htm#:~:text=Each%20spatial%20reference%20includes%20a,)%20and%203857%20%2D%20Web%20Mercator.

File: API.md [hunk 3]
```diff
@@ -50,6 +50,16 @@ echo $geometryCollection->getGeometries()[1]->latitude; // 0
 echo $geometryCollection[1]->latitude; // 0
 ```
 
+## Available Constants
+
+Spatial reference identifiers (SRID) identify the type of coordinate system to use.
```
Comment (MatanYadaev) [inline API.md:55]: Can you please mention that it's about an enum? It seems like this text is a general explanation of SRID, and not explaining that the package suggests using an enum for SRID values.

**Review by MatanYadaev — CHANGES_REQUESTED** [review]
Thanks for PR! :)
I left a few comments.

File: src/Constants/SRID.php (hunk 2 above)
Comment (Riley19280) [reply src/Constants/SRID.php:9]: Yes seems you are correct, I have update the enum to address

File: src/Constants/SRID.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace MatanYadaev\EloquentSpatial\Constants;
+
+/**
```
Comment (MatanYadaev) [inline src/Constants/SRID.php:5]: Can you please remove the doc block? I think it's unnecessary.

File: src/Constants/SRID.php [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+ * Spatial reference identifiers identify the type of coordinate system to use.
+ *
+ * This includes the most common ones listed on wikipedia
+ * Sourced from https://en.wikipedia.org/wiki/Spatial_reference_system
+ */
+enum SRID: int
```
Comment (MatanYadaev) [inline src/Constants/SRID.php:11]: Can we change it to `Srid` instead of `SRID`?

File: src/Constants/SRID.php [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace MatanYadaev\EloquentSpatial\Constants;
```
Comment (MatanYadaev) [inline src/Constants/SRID.php:3]: Can we change it to `Enums` instead of `Constants`?

**Review by MatanYadaev — APPROVED** [review]

**Pull request merged**